app/src/main/java/org/simple/clinic/registration/facility/RegistrationFacilitySelectionScreen.kt

```diff
@@ -68,16 +47,15 @@ class RegistrationFacilitySelectionScreen(
   }
 
   private val delegate by unsafeLazy {
-    val uiRenderer = uiRendererFactory.create(this)
     val screenKey = screenRouter.key<RegistrationFacilitySelectionScreenKey>(this)
 
     MobiusDelegate.forView(
         events = events.ofType(),
         defaultModel = RegistrationFacilitySelectionModel.create(screenKey.ongoingRegistrationEntry),
         update = RegistrationFacilitySelectionUpdate(),
         effectHandler = effectHandlerFactory.create(this).build(),
-        init = RegistrationFacilitySelectionInit.create(config),
-        modelUpdateListener = uiRenderer::render
+        init = RegistrationFacilitySelectionInit(),
+        modelUpdateListener = { /* Nothing to do here */ }
```

Comment:
We can remove the `modelUpdateListener` since it already has a default implementation in `MobiusDelegate`

Reply:
Good point, I'll make this change.